### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by HMU59** [DavidKrGH/BackupScripts#4] [General]
Hello @DavidKrGH 

I just wanted to thank you for all your work, i'm currently setting up a large backup system for multiple Minecraft servers, and your BackupScript is very useful for me !

I hope this will make you happy and motivate you to continue to maintain this script :)

Cheers

(sorry if my english is not perfect, it's not my main language)

**Comment by DavidKrGH** [comment]
Hello @HMU59 ,

Thank you very much for your kind words. I'm pleased that my project is helping you to realise your backups.

Since I use the scripts myself for my backups, the support is secured for now.

Best regards.
